include/edm4hep/Constants.h

```diff
@@ -50,6 +50,14 @@ static constexpr const char* pidParameterNames = "ParameterNames";
 static constexpr const char* pidAlgoName = "AlgoName";
 static constexpr const char* pidAlgoType = "AlgoType";
 
+// Parameter names for Generator level metadata
+static constexpr const char* generatorToolVersionLabel = "generatorToolVersion";
+static constexpr const char* generatorToolNameLabel = "generatorToolName";
+static constexpr const char* generatorToolDescriptionLabel = "generatorToolDescription";
+static constexpr const char* generatorEventParametersLabel = "generatorEventParameters";
+static constexpr const char* generatorPdfInfoLabel = "generatorPdfInfo";
+static constexpr const char* generatorWeightNamesLabel = "generatorWeightNames";
```

Comment:
```suggestion
static constexpr const char* generatorToolVersion = "generatorToolVersion";
static constexpr const char* generatorToolName = "generatorToolName";
static constexpr const char* generatorToolDescription = "generatorToolDescription";
static constexpr const char* generatorEventParameters = "generatorEventParameters";
static constexpr const char* generatorPdfInfo = "generatorPdfInfo";
static constexpr const char* generatorWeightNames = "generatorWeightNames";
```
I would remove the `Label` bit from the name since we don't have that for the other constants either.

Reply:
I would rather add label to the other ones as well

Reply:
At least some of them are already used for quite some time, so changing them would require a bit of work.

Reply:
I'm in favor of removing Label for consistency, that these are strings should something that a modern editor / IDE will tell you already. We also have right now all the constants except one start with upper case both the variable name and the string so maybe we want to keep some sort of style here.

Reply:
I am also in favor of both, i.e. upper case variable names and values and removing the `Label` bit.